### ISSUE TITLE
Random error-based infunctionality of individual players clipboards

### ISSUE THREAD

**Issue by TheoRetisch1** [IntellectualSites/FastAsyncWorldEdit-1.13#159]
When I try to copy something to the clipboard I get the following error message: "The requested operation cannot be performed on a file with a user mapped section open". The clipboard is stored on the disk. The system has permissions to write to the folder. I've checked this and otherwise the problem would also occur with every user. The problem does not occur with every user and does not exist for all users at the same time.  The problem occurs randomly, not reproducible and can only be solved by restarting the server (I tried /fawe reload, the manual deletion of the files and to pos1 + pos2 something else). All non clipboard based operations still work. 
```
clipboard:
  # Store the clipboard on disk instead of memory
  #  - Will be slightly slower
  #  - Uses 2 bytes per block
  use-disk: true
  # Compress the clipboard to reduce the size:
  #  - TODO: Buffered random access with compression is not implemented on disk yet
  #  - 0 = No compression
  #  - 1 = Fast compression
  #  - 2-17 = Slower compression
  compression-level: 0
```
```
_paths:
  tokens: "tokens"
  # Put any minecraft or mod jars for FAWE to be aware of block textures
  textures: "textures"
  heightmap: "heightmap"
  history: "history"
  # Multiple servers can use the same clipboards
  #  - Use a shared directory or NFS/Samba
  clipboard: "clipboard"
  # Each player has their own sub directory for schematics
  per-player-schematics: false
  commands: "commands"
  # Number of days to keep history on disk before deleting it
  delete-after-days: 1_
```

Any ideas what this is related to or what I can try to find the reason for the issue?

error: https://pastebin.com/RKD3NrSZ
FAWE - 1.13.110
Paper 1.13.2 - 626

**Comment by TheoRetisch1** [comment]
Confirmed for version 1.13-117. Very annoying. Any ideas?

**Comment by TheoRetisch1** [comment]
Okay, I was able to reproduce the error with FAWE-163. If you copy something and relog, you get the error. This means, if something is stored in the clipboard, you must not relog! The clipboard file of a player is not always deleted correctly with clearclipboard and cannot be deleted manually because it is open in Java(TM) Platform SE binary.

**Comment by TheoRetisch1** [comment]
If you enter /clearclipboard before the relog, you can delete the file after the relog and use the clipboard afterwards. If you don't enter /clearclipboard before the relog, you can't delete the file anymore and the error can only be fixed by restarting.

**Comment by MattBDev** [comment]
Thanks for the detailed information. It will help narrow down the cause

**Comment by TheoRetisch1** [comment]
Confirmed for the latest build. Very nasty issue. Can I do anything in order to help tracking down the issue?

**Comment by TheoRetisch1** [comment]
What dafuq does that mean "Fixed commanding"? It's not fixed for me in 1.13.204? It's not even in the changelog lol

**Comment by N0tMyFaultOG** [comment]
It means it's fixed in one of the snapshots for 1.14, which is the priority rn.

**Comment by TheoRetisch1** [comment]
Uff ok thank you for the clarification!
